recipes/glu/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+import os
+
+class SysConfigGLUConan(ConanFile):
+    name = "glu"
+    version = "system"
+    description = "cross-platform virtual conan package for the GLU support"
+    topics = ("conan", "opengl", "glu")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.opengl.org/"
+    license = "MIT"
```

Comment:
Wrong license

What license does it have? 🤔

Reply:
Good question, I've copied this from `opengl/system`, assuming that in this case the license pertains to the recipe. The package does not contain any GLU files so I guess `license` should be empty then?

Reply:
I changed `license` to `unspecified`, since GLU header and libs are apparently available under different licenses on different OSes (e. g. the header from Microsoft has different terms than the one from Mesa).